apis/feedapi/feedapi.yaml

```diff
@@ -0,0 +1,117 @@
+swagger: '2.0'
+info:
+  version: '1.0'
+  title: User Feed  API
+  description: >-
+    - The User Feed APIs  are used to store all alerts that could be presented to users. They include alerts sent to users while migrating tenant environments from one channel to another.
```

Comment:
@anmol2302 please check the way this sentence is rephrased. let me know if it is fine.